tests/tests/tier0/bluechictl-set-get-default-target/test_bluechictl_set_get_deault_target.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+
+from bluechi_test.config import BluechiAgentConfig, BluechiControllerConfig
+from bluechi_test.machine import BluechiAgentMachine, BluechiControllerMachine
+from bluechi_test.test import BluechiTest
+
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+    node_foo = nodes[NODE_FOO]
+    multi_user_string = "multi-user.target"
+    graphical_string = "graphical.target"
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.exec_run("systemctl get-default")
+    assert systemd_result in bc_result
+
+    if systemd_result == multi_user_string:
+        ctrl.bluechictl.set_default_target(NODE_FOO, graphical_string, "false")
+        _, systemd_result = node_foo.exec_run("systemctl get-default")
+        assert systemd_result == graphical_string
+    else:
+        ctrl.bluechictl.set_default_target(NODE_FOO, multi_user_string, "false")
+        _, systemd_result = node_foo.exec_run("systemctl get-default")
+        assert systemd_result == multi_user_string
+
+    _, bc_result = ctrl.bluechictl.get_default_target(NODE_FOO)
+    _, systemd_result = node_foo.exec_run("systemctl get-default")
+    assert systemd_result in bc_result
+
+
+def test_bluechi_set_default_target(
```

Comment:
Please align the method name with file name

Reply:
Changed